### PR TITLE
Add Javascript API integrations page

### PR DESCRIPTION
**Pull request by isakkeyten**
I've previously opened up [this](https://github.com/windicss/windicss/issues/293) issue about how to consume the Javascript API of Windi CSS, and was given some good example files.

But I'd like it if they were added to the docs so that other developers won't have to open the same issues again.

Let me know if you would like to change some wording, or I have a misspelling, or anything I need to change.

### CODE REVIEW

File: .vitepress/config/sidebar.ts [hunk 1]
```diff
@@ -148,6 +148,7 @@ export const sidebar: DefaultTheme.Config['sidebar'] = {
         // { text: 'Next.js', link: '/integrations/nextjs' },
         { text: 'Svelte', link: '/integrations/svelte' },
         { text: 'CLI', link: '/integrations/cli' },
+        { text: 'Javascript', link: '/integrations/javascript' },
```
Comment (hannoeru) [inline .vitepress/config/sidebar.ts:151]:
```suggestion
        { text: 'JavaScript API', link: '/integrations/javascript' },
```

File: .vitepress/theme/components/logos/Logo.vue [hunk 2]
```diff
@@ -9,6 +9,7 @@ import VSCode from '/@vite-icons/logos/visual-studio-code'
 import Webstrom from '/@vite-icons/logos/webstorm'
 import PostCSS from '/@vite-icons/logos/postcss'
 import Gridsome from '/@vite-icons/logos/gridsome-icon'
+import Javascript from '/@vite-icons/logos/javascript'
```
Comment (hannoeru) [inline .vitepress/theme/components/logos/Logo.vue:12]:
```suggestion
import JavaScript from '/@vite-icons/logos/javascript'
```

File: .vitepress/theme/components/logos/Logo.vue [hunk 3]
```diff
@@ -45,6 +46,8 @@ const Logo = computed(() => {
       return Webstrom
     case 'postcss':
       return PostCSS
+    case 'javascript':
+      return Javascript
```
Comment (hannoeru) [inline .vitepress/theme/components/logos/Logo.vue:50]:
```suggestion
      return JavaScript
```

File: guide/installation.md [hunk 4]
```diff
@@ -32,6 +32,11 @@ Supports for build tools are framework-agnostic, they work for most frameworks w
     link: '/integrations/cli',
     logo: 'cli',
   },
+    {
+    title: 'Javascript',
+    link: '/integrations/javascript',
+    logo: 'javascript',
+  },
```
Comment (hannoeru) [inline guide/installation.md:39]: I think this should be a new section call `API`

**Review by hannoeru — CHANGES_REQUESTED** [review]
Looks great! just a few changes, also don't forgot to lint your code.

**Comment by isakkeyten** [comment]
> Looks great! just a few changes, also don't forgot to lint your code.

The linter picked up a lot more files than the ones this PR touches, but I discarded those.
Might be a good idea to run it in a separate PR. :)

**Review by hannoeru — APPROVED** [review]

**Pull request merged**